### PR TITLE
Adding grafana dashboards based on prometheus metrics

### PR DESCRIPTION
**Pull request by deimosfr**
Adding grafana dashboards based on promtheus metrics for standard and k8s config

### CODE REVIEW

File: contrib/grafana/traefik-kubernetes.json [hunk 1]
```diff
@@ -0,0 +1,1374 @@
...
+        "y": 17
+      },
+      "id": 15,
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": true,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "prometheus",
```
Comment (mmatur) [inline contrib/grafana/traefik-kubernetes.json]: Could you please replace `"datasource": "prometheus"` by `"datasource": "${DS_PROMETHEUS}"`

File: contrib/grafana/traefik-kubernetes.json [hunk 2]
```diff
@@ -0,0 +1,1374 @@
...
+            "align": false,
+            "alignLevel": null
+          }
+        },
+        {
+          "aliasColors": {},
+          "bars": true,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "prometheus",
```
Comment (mmatur) [inline contrib/grafana/traefik-kubernetes.json]: Could you please replace `"datasource": "prometheus"` by `"datasource": "${DS_PROMETHEUS}"`

File: contrib/grafana/traefik-kubernetes.json [hunk 3]
```diff
@@ -0,0 +1,1374 @@
...
```
Comment (mmatur) [inline contrib/grafana/traefik-kubernetes.json]: Could you please replace `"datasource": "prometheus"` by `"datasource": "${DS_PROMETHEUS}"`

File: contrib/grafana/traefik-kubernetes.json [hunk 4]
```diff
@@ -0,0 +1,1374 @@
...
```
Comment (mmatur) [inline contrib/grafana/traefik-kubernetes.json]: Could you please replace `"datasource": "prometheus"` by `"datasource": "${DS_PROMETHEUS}"`

File: contrib/grafana/traefik.json [hunk 5]
```diff
@@ -0,0 +1,1055 @@
...
+        "y": 17
+      },
+      "id": 15,
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": true,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "prometheus",
```
Comment (mmatur) [inline contrib/grafana/traefik.json]: Could you please replace `"datasource": "prometheus"` by `"datasource": "${DS_PROMETHEUS}"`

File: contrib/grafana/traefik.json [hunk 6]
```diff
@@ -0,0 +1,1055 @@
...
+            "align": false,
+            "alignLevel": null
+          }
+        },
+        {
+          "aliasColors": {},
+          "bars": true,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "prometheus",
```
Comment (mmatur) [inline contrib/grafana/traefik.json]: Could you please replace `"datasource": "prometheus"` by `"datasource": "${DS_PROMETHEUS}"`

File: contrib/grafana/traefik.json [hunk 7]
```diff
@@ -0,0 +1,1055 @@
...
```
Comment (mmatur) [inline contrib/grafana/traefik.json]: Could you please replace `"datasource": "prometheus"` by `"datasource": "${DS_PROMETHEUS}"`

File: contrib/grafana/traefik-kubernetes.json [hunk 8]
```diff
@@ -0,0 +1,1374 @@
...
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "",
+  "title": "Traefik",
+  "uid": "000000113",
```
Comment (mmatur) [inline contrib/grafana/traefik-kubernetes.json]: Could you please replace `000000113` by `TraefikKubernetes`

File: contrib/grafana/traefik-kubernetes.json [hunk 9]
```diff
@@ -0,0 +1,1374 @@
...
+  "version": 4
```
Comment (mmatur) [inline contrib/grafana/traefik-kubernetes.json]: Could you please set version to 1

File: contrib/grafana/traefik.json [hunk 10]
```diff
@@ -0,0 +1,1055 @@
...
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "",
+  "title": "Traefik",
+  "uid": "000000113",
```
Comment (mmatur) [inline contrib/grafana/traefik.json]: Could you please replace `000000113` by `Traefik`

File: contrib/grafana/traefik.json [hunk 11]
```diff
@@ -0,0 +1,1055 @@
...
+  "version": 4
```
Comment (mmatur) [inline contrib/grafana/traefik.json]: Could you please set version to 1

**Comment by deimosfr** [comment]
modifcations done

**Pull request merged**